mpisppy/utils/config.py

```diff
@@ -625,33 +625,46 @@ def gradient_args(self):
                            description="name of the gradient rho file (must be csv)",
                            domain=str,
                            default='')
-        self.add_to_config("order_stat",
-                           description="order statistic for rho (must be between 0 and 1)",
+        self.add_to_config("grad_display_rho",
+                           description="display rho during gradient calscs (default True)",
+                           domain=bool,
+                           default=True)
+        self.add_to_config("grad_primal_thresh",
+                           description="primal threshold for diff during gradient calcs",
+                           domain=float,
+                           default=0.001)
+        self.add_to_config("grad_dual_thresh",
+                           description="dual threshold for abs norm during gradient calcs",
+                           domain=float,
+                           default=0.1)
+        self.add_to_config("grad_pd_thresh",
+                           description="threshold for dual/primal during gradient calcs",
+                           domain=float,
+                           default=0.1)
+        self.add_to_config("grad_order_stat",
+                           description="order statistic for gradient based rho (must be between 0 and 1)",
                            domain=float,
                            default=-1.0)
 
-    def rho_args(self):
-        self.add_to_config("whatpath",
+    def grad_rho_args(self):
+        self.add_to_config("grad_whatpath",
                            description="path to csv file with what",
                            domain=str,
                            default='')
-        self.add_to_config("rho_file",
-                           description="name of the rho file (must be csv)",
-                           domain=str,
-                           default='')
-        self.add_to_config('rho_setter',
+        self.add_to_config('grad_rho_setter',
                            description="use rho setter from a rho file",
                            domain=bool,
                            default=False)
-        self.add_to_config("rho_path",
-                           description="csv file for the rho setter",
+        
+        self.add_to_config("grad_rho_path",
+                           description="csv file for the the grad based rho file (???)",
```

Comment:
Why the "???"?  - Also, fix "the the". Also, it's a directory - no (as in no file)?

Reply:
The ??? is to remind me that I need to find out what actually goes in this file and document it in this help string